abdiff/config.py

```diff
@@ -31,6 +33,18 @@ def webapp_port(self) -> int:
         port = self.WEBAPP_PORT or "5000"
         return int(port)
 
+    @property
+    def transmogrifier_max_workers(self) -> int:
+        """Maximum number of Transmogrifier containers to run in parallel."""
```

Comment:
"in parallel" -> "concurrently" / "asynchronously"?

Reply:
Same here I think: https://github.com/MITLibraries/transmogrifier-ab-diff/pull/37#discussion_r1816942118